docs/eventing/samples/kafka/channel/README.md

```diff
@@ -151,3 +151,74 @@ Now you can see the events in the log of the `ksvc` using the command:
 ```
 kubectl logs --selector='serving.knative.dev/service=broker-kafka-display' -c user-container
 ```
+
+## Authentication against an Apache Kafka
+
+In production enviornments it is common that the Apache Kafka cluster is secured using [TLS](http://kafka.apache.org/documentation/#security_ssl) or [SASL](http://kafka.apache.org/documentation/#security_sasl). This section show how to confiugure the `KafkaChannel` to work against a protected Apache Kafka cluster, with the two supported TLS and SASL authentication methods.
+
+### TLS authentication
+
+To use TLS authentication you must create:
+
+* A CA certificate
+* A client certificate and key
+
+**NOTE:** Kafka channels require these files to be in `.pem` format. If your files are in a different format, you must convert them to `.pem`.
+
+
+1. Create the certificate files as secrets in your chosen namespace:
+```
+$ kubectl create secret --namespace <namespace> generic <kafka-auth-secret> \
+  --from-file=ca.crt=caroot.pem \
+  --from-file=user.crt=certificate.pem \
+  --from-file=user.key=key.pem
+ ```
+
+*NOTE:* It is important to use the same keys (`ca.crt`, `user.crt` and `user.key`).
+
+Now all you need is to reference your secret and its namespace on the `config-kafka` ConfigMap:
```

Comment:
Same comment as "Now all you need...." in other section below. Make it a step 2 without the fluff.